### PR TITLE
Exit if template input file does not exist

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

File: usine.py [hunk 1]
```diff
@@ -302,10 +305,13 @@ def _call_command(self, cmd, **kwargs):
                      channel.recv_exit_status())
         channel.close()
         if ret.code:
-            print(red(ret.stderr))
-            sys.exit(ret.code)
+            self.exit(ret.stderr, ret.code)
         return ret
 
+    def exit(self, msg, code=1):
```
Comment (davidbgk) [inline usine.py:311]: I would have named that one `error` given that we only want to `exit` on errors?

**Review by davidbgk — APPROVED** [review]

File: usine.py (hunk 1 above)
Comment (yohanboniface) [reply usine.py:311]: You mean naming the function or the `msg` parameter?

Comment (davidbgk) [reply usine.py:311]: The naming

Comment (yohanboniface) [reply usine.py:311]: Humm, I'm not sure to agree then ;)
I think `exit` explains more what the function does: it *exits* the script (in the same meaning as `sys.exit`).

Comment (davidbgk) [reply usine.py:311]: Alright, go for it then 👍

**Pull request merged**